modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -16,6 +16,9 @@ Upgrades to cluster-level and node configuration components are performed using
 
 The latest versions of each downloadable package are found in the *Downloads* page, located in Anypoint Runtime Manager in the *Runtime Fabrics* tab. A link to each package’s release notes is provided to show the changes.
 
+[IMPORTANT]
+You must be running Runtime Fabric version 1.4.54 in order to upgrade to Runtime Fabric version 1.5 or later. If you are using an earlier version than Runtime Fabric version 1.4.54, you must upgrade to Runtime Fabric version 1.4.54 before upgrading to version 1.5 or later.
```

Comment:
```suggestion
You must be running Runtime Fabric version 1.4.54 to upgrade to version 1.5 or later. If you are using a version earlier than 1.4.54, you must upgrade to 1.4.54 before upgrading to 1.5 or later.
```
Don't think we need to keep repeating "version" and the product name in this short context. Also think we could run with this:
```suggestion
To upgrade to Runtime Fabric version 1.5 or later, you must either already be running version 1.4.54 or first upgrade to 1.4.54 before upgrading to 1.5.
```